runtime/runtime/report/json/application.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2024, ASAM e.V.
+#
+# This Source Code Form is subject to the terms of the Mozilla
+# Public License, v. 2.0. If a copy of the MPL was not distributed
+# with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+from .constants import Constants
+from .json_formatter import JsonFormatter
+
+
+class Application:
```

Comment:
Minor: If a class contains only `classmethod`, we might consider removing the class and just using functions.

In `__main__.py`, we can have:

```python
import application

application.export_default_config()
...
```

Reply:
I reviewed the code based upon suggestion. The idea is to have base classes to rapidly develop formatter cliu application.

This has been implemented in `runtime.report.base` under the classes `ReportFormatter` and `ReportApplication`. The json report tool has been implemented with the approach. There is also an initial implementation of the text report tool, even if implementation details are missing (output only "not yet implemented").

The approach introduces some additional features:

 - cli arguments for parameters to override configuration file or create custom configuration
 - supports generation reading result from stdin
 - supports output of the report in stdout or stderr

Example of usage of the new features:

```
cat Result.xqar | qc_report_json -r --param-OutputFile __stdout__ --param-InputFile __stdin__ --param-Indent 4
```